packages/block-library/src/query/index.php

```diff
@@ -123,3 +123,87 @@ function register_block_core_query() {
 	);
 }
 add_action( 'init', 'register_block_core_query' );
+
+/**
+ * Traverse the tree of blocks looking for any plugin block (i.e., a block from
+ * an installed plugin) inside a Query block with the enhanced pagination
+ * enabled. If at least one is found, the enhanced pagination is effectively
+ * disabled to prevent any potential incompatibilities.
+ *
+ * @since 6.4.0
+ *
+ * @param array $parsed_block The block being rendered.
+ *
+ * @return string Returns the parsed block, unmodified.
```

Comment:
```suggestion
 * @param array $parsed_block The block being rendered.
 * @return string Returns the parsed block, unmodified.
```

Nit: WPCS are not to include an empty line between params and returns.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/55714/commits/221b70b90d2eaddb216886fe8d888251679beb0a.